api/api_test.go

```diff
@@ -1,16 +1,17 @@
 // Copyright (c) 2023-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-package main
+package api
 
+/*
```

Comment:
Is this commented out for any reason? if the tests are not working, maybe we should put a skip on them instead of commenting them out.